src/NuGetSdkResolver/NuGetSdkResolver.cs

```diff
@@ -203,8 +203,23 @@ private static bool TryGetMSBuildSdkPackageInfo(FallbackPackagePathResolver fall
                     return false;
                 }
 
-                // Get the installed path and add the expected "Sdk" folder
-                installedPath = Path.Combine(packageInfo.PathResolver.GetInstallPath(packageInfo.Id, packageInfo.Version), "Sdk");
+                if (NativeMethodsShared.IsWindows)
+                {
+                    // Get the installed path and add the expected "Sdk" folder.  Windows file systems are not case sensitive
+                    installedPath = Path.Combine(packageInfo.PathResolver.GetInstallPath(packageInfo.Id, packageInfo.Version), "Sdk");
```

Comment:
Move this outside the `if` and just do if !windows && !exist? seems cleaner to read.

Reply:
👍 I messed with the logic so much I forgot to go back and simplify.  I ❤️ code reviews